open-craft/opencraft
# SE-2108 Fix appservers/vms dns records clean up

This PR removes the DNS records for the appservers of an instance that is being archived.

**Testing Instructions:**
1. Create a new instance and spawn new appservers.
2. Activate the new appservers.
3. Verify the corresponding vm dns records are shown in the dns console.
4. Archive the created instance with active appservers.
5. Verify that the vm dns records are removed.

**Reviewers:**
